src/_style.scss

```
@@ -296,8 +296,15 @@ select.form-control {
 .tooltip-shadow {
   box-shadow: 0px 5px 15px 0px rgba(0,0,0,0.3) !important;
 }
+
+.password-tooltip {
+  line-height: 2;
+  @extend .x-small
+}
+
 #password-requirement-left {
   opacity: 1;
+  @extend .password-tooltip;
```

no need for additional class just do this
```suggestion
  @extend .x-small;
```